dimod/sampleset.py

```diff
@@ -1083,6 +1083,55 @@ def append_variables(self, samples_like, sort_labels=True):
                                        sort_labels=sort_labels,
                                        **self.data_vectors)
 
+    def lowest(self, rtol=1.e-5, atol=1.e-8):
+        """Return a SampleSet containing the samples with the lowest energy.
+
+        Args:
+            rtol (float, optional, default=1.e-5):
+                The relative tolerance. See :func:`numpy.isclose`.
+
+            atol (float, optional, default=1.e-8):
+                The absolute tolerance. See :func:`numpy.isclose`.
+
+        Returns:
+            :obj:`.SampleSet`: A new sample set containing the samples with an
+            energy within tolerance of the minimum energy of the sample set.
+
```

Comment:
How about something like, "A new SampleSet of the lowest-energy samples as delimited by configured tolerances from the lowest-energy sample in the current SampleSet"?